### ISSUE TITLE
Audit addon file names

### ISSUE THREAD

**Issue by kcoleman731** [shapirov103/cdk-eks-blueprint#29]
Rename Addon .ts files for consistency. I suggest either addon.ts or index.ts.

**Comment by shapirov103** [comment]
do you suggest renaming all addon files to be addon.ts/index.ts? Does not sound like the most pressing thing, I can see benefits, such as consistency.

**Comment by shapirov103** [comment]
@parkand1 can you take a look as a training task to debug the development process?  We will need the following:
1. rename all files on a branch
2. update docs/internal-readme.md with clear rules on naming
3. Create a PR to the main and me (and possibly Kevin) as approvers

**Comment by kcoleman731** [comment]
Yep @shapirov103. If we leverage helm charts, we likely will only need a single file in each addon dir. I'm not too concerned about the actual name, just that we are consistent.
